bot/admin/server/src/main/kotlin/BotAdminVerticle.kt

```diff
@@ -105,7 +105,7 @@ open class BotAdminVerticle : AdminVerticle() {
             }
         }
 
-        blockingJsonPost("/analytics/users", botUser) { context, query: UserSearchQuery ->
+        blockingJsonPost("/analytics/users1", botUser) { context, query: UserSearchQuery ->
```

Comment:
Why users1 ?

Reply:
@vsct-jburet This API is not used, I forgot to remove it.
now it's removed.